tests/ui/async-await/large_moves.rs

```diff
@@ -1,10 +1,10 @@
 #![deny(large_assignments)]
-#![feature(large_assignments)]
+#![cfg_attr(attribute, feature(large_assignments))]
 #![cfg_attr(attribute, move_size_limit = "1000")]
 // build-fail
 // only-x86_64
-// revisions: attribute option
-// [option]compile-flags: -Zmove-size-limit=1000
+// revisions: attribute option nothing
+// [option]compile-flags: -Zmove-size-limit=2000
```

Comment:
Please change this in a separate commit so I can see the changes from the code changes separately from the test changes

Reply:
Absolutely. I have now split this PR up into these three commits:

* 2505a7d41e1 Add regression tests for new default `move_size_limit = 4096`
* a9cb3509234 Set default `move_size_limit` to 4kB for `large_assignments` lint on nightly
* 39a42c32880 Mark `#[allow(large_assignments)]` for existing large moves